jetty-home/src/main/resources/bin/jetty.sh

```diff
@@ -531,38 +589,27 @@ case "$ACTION" in
       exit
     fi
 
-    if ! touch "$JETTY_PID"
-    then
-      echo "** ERROR: Unable to touch file: $JETTY_PID"
-      echo "          Correct issues preventing use of \$JETTY_PID and try again."
-      exit 1
-    fi
-
-    if ! touch "$JETTY_STATE"
-    then
-      echo "** ERROR: Unable to touch file: $JETTY_STATE"
-      echo "          Correct issues preventing use of \$JETTY_STATE and try again."
-      exit 1
-    fi
+    testFileSystemPermissions
 
     echo -n "Starting Jetty: "
 
     # Startup from a service file
-    if [ $UID -eq 0 ] && type start-stop-daemon > /dev/null 2>&1
+    if (( USE_START_STOP_DAEMON ))
     then
       unset CH_USER
       if [ -n "$JETTY_USER" ]
       then
         CH_USER="--chuid $JETTY_USER"
       fi
 
-      echo ${RUN_ARGS[@]} --start-log-file="$JETTY_START_LOG" | xargs start-stop-daemon \
+      # use of --pidfile /dev/null disables internal pidfile
+      # management of the start-stop-daemon (see man page)
+      echo ${RUN_ARGS[@]} | xargs start-stop-daemon \
```

Comment:
I'm thinking that the whole ${RUN_ARGS[@]} thing just doesn't work.  We abandonned that approach in the docker shell and instead use files and a dot command.  However, here I think we could perhaps just do:
```suggestion
      { echo jetty.state=$JETTY_STATE;  echo jetty.pid=$JETTY_PID;  echo  $JETTY_SYS_PROPS; "$JAVA" -jar "$JETTY_START" --dry-run=opts,path,main,args ;  } | xargs start-stop-daemon \
```

Reply:
OK I'm missing some complexities with JETTY_ARGS and JETTY_CONF here, but the point is the same: using xargs appears to be a workable way of assembling a command line that correctly preserves quoting.

Reply:
or build a file that we can run with .

Reply:
Actually I think the `echo ${RUN_ARGS[@]} | xargs start-stop-daemon ` mechanism works/